tools/build_config/build.sh

```diff
@@ -85,6 +85,41 @@ function run_package() {
   echo_and_run "cpack -G TGZ"
 }
 
+function build_docs() {
+    # Update release numbers
+    build_id=$(sed -nr '/CPACK_PACKAGE_FILE_NAME/{s/.*"Horace-([^"]+)".*/\\1/p};' ./build/CPackConfig.cmake)
+    sed -i -r 's/release = .*/release = "'${build_id}'"/' ./documentation/user_docs/docs/conf.py
```

Comment:
can `conf.py` be updated pull this data from file rather than performing a temporary `sed` update?

Reply:
I think so, it is basically a python script, I'll have a play, but it will require users to have used the CMake pathway.